RIOT-OS/RIOT
# drivers/{d,e,f}*: add Kconfig modules

### Contribution description
This models more drivers in Kconfig and adds the configuration to their test applications. The modelled drivers are the following:
- dcf77
- dfplayer
- dht
- ds1307
- ds18
- ds3231
- ds3234
- ds75lx
- dsp0401
- dynamixel
- edbg_eui
- feetech
- fxos8700

Network devices were not included so far.

### Testing procedure
- Check that the dependencies make sense
- Check that all the applications have been added to the CI
- Green CI

### Issues/PRs references
None
